packages/styles/scss/components/feature-card/_feature-card.scss

```diff
@@ -70,6 +70,27 @@
 
     @include carbon--breakpoint-down('md') {
       padding-top: aspect-ratio(1, 1);
+
+      .#{$prefix}--card__content {
+        padding: 0;
+        justify-content: space-between;
+      }
+
+      #{$prefix}--card__heading {
+        background-color: $ui-01;
+        margin: 0;
+        width: 100%;
+        padding: carbon--mini-units(2);
+        padding-right: 10%;
+        padding-bottom: $layout-03;
+      }
+
+      #{$prefix}--card__footer {
+        background-color: $ui-01;
```

Comment:
Thanks for the comments @photodow. I was testing this because the way the card structure was setup. Not sure if I'm looking at it the correctly but the anchor link (which holds most of the content) has an absolute position. As the text gets longer, this doesn't allow the heading to increase the height of the card in smaller resolutions, causing the overflow problem mentioned in this issue. But I agree, this solution is too messy so I'm working on a new one.

![image](https://user-images.githubusercontent.com/4054756/111511416-5334d880-8714-11eb-8c1b-5f1d8d6c09ae.png)

Reply:
Hello @photodow, I was able to solve the overflow bug without any background resets or color rules in general.